lib/ansible/modules/remote_management/ucs/ucs_san_connectivity.py

```diff
@@ -0,0 +1,219 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: ucs_san_connectivity
+short_description: Configures SAN Connectivity Policies on Cisco UCS Manager
+description:
+- Configures SAN Connectivity Policies on Cisco UCS Manager.
+- Examples can be used with the UCS Platform Emulator U(https://communities.cisco.com/ucspe).
+extends_documentation_fragment: ucs
+options:
+  state:
+    description:
+    - If C(present), will verify SAN Connectivity Policies are present and will create if needed.
+    - If C(absent), will verify SAN Connectivity Policies are absent and will delete if needed.
+    choices: [present, absent]
+    default: present
+  san_connectivity_list:
+    description:
+    - List of SAN Connectivity Policies.  Allows multiple resource updates with a single UCSM connection.
+    - Either san_connectivity_list or name is required (user can specify either a list or single resource)
+    - See descriptions for name and other options for information on specifying each list element
+  name:
+    description:
+    - Name of the SAN Connectivity Policy
+    - If specifying a single SAN Connectivity Policy, name is required
+  descr:
```

Comment:
For ACI we opted to use `description` as default, and made an alias for `descr`. Since otherwise it looks a bit awkward (and core devs have questioned it too). But it's up to you.

Reply:
Spelled out in the latest commit